template.yaml

```diff
@@ -181,8 +220,6 @@ Resources:
     Properties:
       Handler: no.sikt.nva.pubchannels.handler.fetch.series.FetchSeriesByIdentifierAndYearHandler::handleRequest
       Role: !GetAtt LambdaRole.Arn
```

Comment:
I would suggest keeping the Policies: and create a new managed policy for the dynamodb. It is very tricky to review and remove access from these common roles that we use all over the place.

Reply:
and make it more specific for each table and bucket

Reply:
Splitted everything to policies and attached policies to lambdas